flocx_market/objects/base.py

```diff
@@ -0,0 +1,38 @@
+from oslo_log import log
+from oslo_versionedobjects import base as object_base
+from flocx_market.objects import fields as object_fields
+import datetime
+
+LOG = log.getLogger(__name__)
+
+
+class MarketPlaceObject(object_base.VersionedObject):
+    OBJ_SERIAL_NAMESPACE = 'markete_place_object'
+    OBJ_PROJECT_NAMESPACE = 'markete_place'
+
+    fields = {
+        'created_at': object_fields.DateTimeField(nullable=True),
+        'updated_at': object_fields.DateTimeField(nullable=True),
+    }
```

Comment:
Take this out for now - this only works if we use the TimestampMixin in the database model (which we don't right now).